source/plugins/projects/projects/index.js

```diff
@@ -491,7 +491,7 @@ let projects = {
 	 * 
 	 * @example
 	 * 
-	 * importProject('MyNewProject', '.zip');
+	 * importProject(project, '.zip');
```

Comment:
This does not seem right, `importProject` takes 3 parameters while in the documentation there are only 2.